pkg/model/openstackmodel/firewall.go

```diff
@@ -294,6 +294,26 @@ func (b *FirewallModelBuilder) addKubeletRules(c *fi.ModelBuilderContext, sgMap
 	return nil
 }
 
+// addNodeExporterRules - Allow 9100 TCP port from nodesg
```

Comment:
I don't believe that Kops can be responsible for deploying prometheus, or prometheus-operator.  Kops cant be responsible for the security groups of applications it's not also responsible for managing.

I think we can use additionalSecurityGroups as defined in the instance group spec to enable this, though it seems more manual.  Additionally we might provide a way to declare in the spec that you need additional security groups created.

Reply:
this same deployment (kube-prometheus) works fine in aws kops clusters. @chrisz100 @mikesplain do you have idea should we allow this or not? Should openstack behave in similar way than aws?

Reply:
I actually need this too.  Still interested in thoughts from @chrisz100  and @mikesplain, however I think I will approve this for now and come up with any other solutions I feel might be more appropriate as needed.

We still have the flexibility of changing things as this is still labeled alpha.

/lgtm